README.md

```diff
@@ -20,8 +23,7 @@ You can install `stellarphot` with either `pip` or `conda`.  If you are interest
 
```

Comment:
We probably want to state a minimum requirement of Python 3.11 and also state they need `jupyter`, or does `stellarphot` conda installation handle those requirements.  I think the Python one is critical to mention at least.

Reply:
I think we currently work with Python 3.10, and should be good for Python 3.12 now that astroscrappy has been released for Python 3.12.

Right now the version is enforced in `pyproject.toml` here: https://github.com/feder-observatory/stellarphot/blob/main/pyproject.toml#L11

I don't think we need to bump the minimum to 3.11 yet....

Reply:
jupyterlab is automatically installed because we list ipywidgets as a dependency (at least I *think* that is what is pulling in jupyterlab? maybe it is jupyter-app-launcher that pulls it in?)